generators/docker-compose/index.js

```diff
@@ -0,0 +1,264 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+var shelljs = require('shelljs');
+var crypto = require('crypto');
+var _ = require('lodash');
+
+module.exports = yeoman.generators.Base.extend({
+    initializing: {
+        printJHipsterLogo: function() {
```

Comment:
remove the logo, we dont use the logo on sub generators just put some thing like `generating docker Files for your apps`